src/Notifications/AbstractNotifier.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Dovu\GuardianPhpSdk\Notifications;
+
+use Exception;
+
+abstract class AbstractNotifier
+{
+    public static function getNotifier($notification): AbstractNotifier
+    {
+        $notifier = array_keys($notification)[0];
```

Comment:
Is there ever a state where the notification array is empty?

Reply:
Good catch, thanks ! Added Exception.